config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   devise_for :admin_users, ActiveAdmin::Devise.config
   ActiveAdmin.routes(self)
+  constraints CanAccessFlipperUI do
+    mount Flipper::UI.app(Flipper) => '/flipper'
```

Comment:
What do you think about changing the route to `/feature-flags` or similar so that we make it more friendly to admins? They don't need to know that the tool is called Flipper